### PR TITLE
720 Jump Menu Accessibility issue on LHS

### LINKED ISSUE #720: [Product] Desktop and Mobile Jump Menu Accessibility issue on LHS
Mobile Accessibility Issue:
https://pagespeed.web.dev/analysis/https-lifesciences-danaher-com-us-en-products-family-axon-digidata-1550b-low-noise-data-acquisition-system-plus-humsilencer-html/695awlv6jv?form_factor=mobile

Desktop Accessibility Issue:
https://pagespeed.web.dev/analysis/https-lifesciences-danaher-com-us-en-products-family-axon-digidata-1550b-low-noise-data-acquisition-system-plus-humsilencer-html/695awlv6jv?form_factor=desktop

Mobile - Add aria-label="Specifications" or what the title is

Desktop - Add aria-label="Specifications" or what the title is

### PR DESCRIPTION
**Pull request by sharanagoudapatil**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #720 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://720-accessibility-issue-on-lhs--danaher-ls-aem--hlxsites.hlx.page/us/en/products/family/biozen-biologics-lc-columns

### CODE REVIEW

**Review by duynguyen — APPROVED** [review]
thanks Sharan. only minor comment.
also, will you tackle the "Heading elements are not in a sequentially-descending order" and "Background and foreground colours do not have a sufficient contrast ratio." in another PR?

File: blocks/page-tabs/page-tabs.js [hunk 1]
```diff
@@ -174,7 +176,7 @@ export default async function decorate(block) {
   const selectEl = document.getElementById('selectedTab');
   selectEl.addEventListener('change', (event) => {
     const innerText = event.target.value;
-    window.location.hash = `#${innerText.toLowerCase()}`;
```
[the remark this answers is not in the document]
Comment (sharanagoudapatil) [reply blocks/page-tabs/page-tabs.js:177]: done.

**Comment by sharanagoudapatil** [comment]
> thanks Sharan. only minor comment. also, will you tackle the "Heading elements are not in a sequentially-descending order" and "Background and foreground colours do not have a sufficient contrast ratio." in another PR?

@duynguyen - Thanks for approving PR. 
The above changes will be taken care as part of another PR. I'll check with @dave-nichols on the same. Thanks.

**Review by davenichols-DHLS — APPROVED** [review]
Approved, thanks Sharan

**Pull request merged**